plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/util/TestPrioritizedFifoExecutor.java

```diff
@@ -70,7 +70,8 @@ public void testCounter()
             futures.add(executor.submit(() -> {
                 try {
                     // wait for the go signal
-                    awaitUninterruptibly(startLatch, 1, TimeUnit.MINUTES);
+                    @SuppressWarnings("CheckReturnValue")
+                    boolean ignored = awaitUninterruptibly(startLatch, 1, TimeUnit.MINUTES);
```

Comment:
why ignore?

Reply:
To leave the behavior as it was before.

Reply:
this is a test, so it would not be unsafe to just improve it